vonk/configuration/appsettings.rst

```diff
@@ -346,6 +346,7 @@ Be aware that:
 
 * support for _type and _id cannot be disabled
 * the Administration API requires support for the 'url' SearchParameter on the conformance resourcetypes
+* this uses the search parameter names, not the path within the resource - so for example to specify `Patient.address.postalCode <http://hl7.org/fhir/R4/patient.html#search>`_ as a supported parameter, you'd use ``"Patient.address-postalcode"``.
```

Comment:
Could you change the 'as supported parameter' to 'as supported expression' or something indicating that you mean the element/field within the resource? Because now it is confusing what is meant by parameter.

Reply:
I've changed it to 'as a supported location' - because while technically it's a fhirpath expression, FHIR newbies won't be thinking of it as such, I think a location is more natural to them. What do you think?

Reply:
Yes, I think that's better than expression.